hadoop-hdds/server-scm/src/main/java/org/apache/hadoop/hdds/scm/server/SCMClientProtocolServer.java

```diff
@@ -732,6 +733,12 @@ public boolean getReplicationManagerStatus() {
     return scm.getReplicationManager().isRunning();
   }
 
+  @Override
+  public ReplicationManagerReport getReplicationManagerReport() {
+    AUDIT.logWriteSuccess(buildAuditMessageForSuccess(
+        SCMAction.GET_REPLICATION_MANAGER_REPORT, null));
+    return scm.getReplicationManager().getContainerReport();
+  }
```

Comment:
Shouldn't we require admin privilege for this operation?

Reply:
I'm not sure. The information provided by the report is not sensitive - just a cluster health report and it is not resource intensive to produce it.

Looking at the other operations in that same protocol server.

start/stop replication manager are admin ops. Viewing its status is not an admin op.

Start / stop balance is an admin op. Viewing its status is not.

I am happy to make this one an admin op, but as it is read only and not performance intensive so I don't know if it needs to be.

Reply:
> I don't know if it needs to be.

I don't know either.  But it's easier to start out with more restriction and relax it later, than to realize we should have limited access in the previous release. ;)

What do you think @swagle?

Reply:
Looking at this from CLI point of view, status generally has the same privilege as start/stop because it might be accessing resources like opening fds or checking ports for example to determine status. Although we do not need to do such things, use expectation is that it is privileged IMO.

Reply:
OK - I have pushed a new commit to make it an admin operation. The existing operations seem a bit inconsistent on what requires admin privs or not.